cli/src/common_utils/lr-serial-number

```diff
@@ -0,0 +1,22 @@
+if [[ $# -eq 0 ]]; then
```

Comment:
1.  Add shbang
1.  Convert this to function and call the same function end of file.